api_app/serializers.py

```diff
@@ -179,10 +194,48 @@ def validate(self, attrs: dict) -> dict:
         file_obj.seek(0)
         file_buffer = file_obj.read()
         attrs["md5"] = calculate_md5(file_buffer)
+        self.filter_analyzers_and_connectors(attrs)
         logger.debug(f"after attrs: {attrs}")
         return attrs
 
 
+class MultipleObservableAnalysisSerializer(rfs.ListSerializer):
+    """
+    ``Job`` model's serializer for Multiple Observable Analysis.
+    Used for ``create()``.
+    """
+
+    def update(self, instance, validated_data):
+        raise NotImplementedError("This serializer does not support update().")
+
+    child = None
+    many = True
+    observables = rfs.ListField(required=True)
+
+    def to_internal_value(self, data):
+        ret = []
+        errors = []
+
+        common_data = dict(data)
+        common_data.pop("observables", None)
+        for classification, name in data.get("observables"):
+            item = copy.deepcopy(common_data)
+            item["observable_name"] = name
+            item["observable_classification"] = classification
+            try:
+                validated = self.child.run_validation(item)
+            except ValidationError as exc:
+                errors.append(exc.detail)
+            else:
+                ret.append(validated)
+                errors.append({})
+
+        if any(errors):
```

Comment:
The request would not fail if additional analyzers are passed. You can check the sample request and response I've posted in the PR comments

Reply:
yeah but I was mentioning the entire observable. I mean, in your example both observables are analyzed cause there is at least one analyzer that works for them. In the case those analyzers would be 0, right now the entire multi-analysis fails. 
So my question is: if one *observable* does not pass `ValidationError`, should we drop the request completely or we should drop the request only if *all* the observables do not pass validation?

I think that it makes sense what you did but I thought this could be something to discuss

Reply:
Oh. Got it. I'd say in that case it's better to drop the request as accepting requests partially could lead to a lot of confusion, especially for large requests

Reply:
agree